cast/src/lib.rs

```diff
@@ -676,8 +676,13 @@ where
             .ok_or_else(|| eyre::eyre!("tx not found: {:?}", tx_hash))?;
 
         Ok(if let Some(ref field) = field {
-            get_pretty_tx_attr(&tx, field)
-                .ok_or_else(|| eyre::eyre!("invalid tx field: {}", field))?
+            match field.as_ref() {
+                "raw" => {
+                    format!("0x{}", hex::encode(tx.rlp()))
+                }
+                _ => get_pretty_tx_attr(&tx, field)
+                    .ok_or_else(|| eyre::eyre!("invalid tx field: {}", field.to_string()))?,
```

Comment:
we should document this

I kinda like this, we could alternative also convert the field argument into an enum `Raw, Field(String)`

but this seems fine if documented

Reply:
+1 to the enum—that'd be really nice! but also fine w/ me if documented